lib/endpoint/tags/table.js

```diff
@@ -126,6 +126,47 @@ function rowToParameter (i, el) {
     params.type = 'objects[]'
   }
 
+  if (params.name === 'content') {
+    if (/reaction type/.test(params.description) === true) {
+      params.enum = ['+1',
+        '-1',
+        'laugh',
+        'confused',
+        'heart',
+        'hooray']
+    }
+  }
+
+  if (params.name === 'type') {
+    if (/Normally this is a `commit`/.test(params.description) === true) {
+      params.enum = ['commit',
+        'tree',
+        'blob']
+    }
+  }
+
+  if (params.name === 'state') {
+    if (/active/.test(params.description) === true) {
+      params.enum = ['active']
+    }
+  }
+
+  if (params.name === 'default_repository_permission') {
+    if (/organization repositories:/.test(params.description) === true) {
```

Comment:
I feel like for this one we should try to match the pattern which looks to me like

```
\n* `<parameter>` - <description>
```

Reply:
I think for this `Default permission level members have for organization repositories:` would work. Is it ok?

Reply:
okay, good enough for now :). I wonder if we have this pattern in other places

```
\n* `<parameter>` - <description>
```

If we do, we should remove this workaround and implement more flexible pattern matching.